catalog/ui/catalog-ui-search/pom.xml

```diff
@@ -341,6 +346,7 @@
                             javax.servlet.http;version="[3,4)",
                             javax.servlet;version="[3,4)",
                             org.apache.commons.logging; version="[1.1.0,2.0.0)",
+                            org.codice.ddf.catalog.ui.metacard.workspace.transformer,
```

Comment:
does this need to be explicitly imported?

Reply:
Yes. `catalog-ui-search` fails without it.

Reply:
[See [my previous comment](https://github.com/codice/ddf/pull/3271#discussion_r195489203).]